extensions/wikia/AdEngine/js/slot/revcontentSlots.js

```diff
@@ -1,10 +1,10 @@
 /*global define*/
 define('ext.wikia.adEngine.slot.revcontentSlots', [
```

Comment:
What do you think about removing whole revcontent? Worth to ask Nef/Nandy and make some small clean up :)